medusa/providers/nzb/binsearch.py

```diff
@@ -121,7 +121,7 @@ def parse(self, data, mode):
                     title_re = title_regex.search(title)
                     # Try and get the the article subject from the wierd binsearch format
                     if title_re:
-                        title = title_re.group(1) or title_re.group(2) or attributes[2].find("span").get_text()
+                        title = title_re.group(1) if len(title_re.group(1)) > 3 or title_re.group(2) or attributes[2].find("span").get_text()
```

Comment:
At this point this should probably be broken down into either separate steps or an if statement as its getting a little long